bfsbdump

```diff
@@ -53,6 +53,15 @@ def GetPlatformName():
 def CountSetBits(n):
     return bin(n).count("1")
 
+def GetEcid():
+    str = os.popen('xxd -p /sys/firmware/acpi/tables/SSDT1 2>/dev/null | tr -d "\\n"').readline()
+    pos1 = str.find('656369643031000e')
+    pos2 = str.find('656369643233000e')
+    if (pos1 >= 0) and (pos2 >= 0):
+        return (str[pos1+16:pos1+32]) + str[pos2+16:pos2+32]
+    else:
+        return ""
+
 class ChipSecureBootStatus:
 
     def __init__(self):
```

Comment:
I suggest we add the ECID as an Object attribute.

`
def __init__(self):
        self.PlatformName = GetPlatformName()
        self.ecid = GetEcid()
`

Reply:
done